icons/supabase.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase icon</title><path d="M9.90318 23.12655c0 .48238.39086.87345.8731.87345.25854 0 .49092-.11244.65073-.29115l.03443-.03876 6.67515-9.02463h-6.6005L9.93482 22.8867l-.0043.0221a.87306.87306 0 00-.02736.21776z c0 .48238.39086.87345.8731.87345.25854 0 .49092-.11244.65073-.29115l.03443-.03876 6.67515-9.02463h-6.6005L9.93482 22.8867l-.0043.0221a.87306.87306 0 00-.02736.21776zm10.66825-12.23692c0-.77185-.62547-1.39753-1.39697-1.39753h-6.63708l-1.00129 5.15336h6.6005l2.12724-2.87601.08487-.12245c.14093-.21817.22272-.47825.22272-.75737zM3.42857 13.248c0 .77178.62544 1.39746 1.39695 1.39746H11.53612L12.5374 9.4921H5.85483l-2.02488 2.7376-.1304.19096c-.17035.2316-.27098.5177-.27098.82733zM14.19771.87345c0-.48239-.39092-.87345-.8731-.87345-.23978 0-.45697.0967-.61476.25324L12.5886.3882 5.85481 9.4921h6.68257l1.62675-8.37218.02967-.16354a.87447.87447 0 00.0039-.08292z"/></svg>
```

Comment:
Thanks for submitting this @adamalston!
Looks like your PR version (red) is actually a fair bit thinner than the source icon (black). There's also a load of excess points showing up - would you mind looking to simplify this?
![image](https://user-images.githubusercontent.com/1830867/106272662-4d00a080-6229-11eb-972a-49714b21e859.png)


Is there also any chance you can get this down to 3 or 4 decimals of precision? While 5 passes the linter, we try and keep it to 3/4 decimals where possible.